text/0118-search-api.md

```diff
@@ -1152,16 +1155,29 @@ Added to the search response when `facets` is set for a search query. It contain
 
 If a field distributed as a facet contains no value, it is returned as a `facetDistribution` field with an empty object as value.
 
-> See [3.1.4. `facet`](#314-facets) section.
+> See [3.1.4. `facets`](#314-facets) section.
+
+#### 3.2.10. `facetStats`
+
+- Type: Object
+- Required: False
+
+When using the `facets` parameter, the distributed facets that contain some numeric values are displayed in a `facetStats` object that contains, per facet, the numeric `min` and `max` values of the hits returned by the search query.
```

Comment:
Do they represent the min and max value of the `hits` in `{ hits: [] }` or the min-max value of the remaining candidates?

Reply:
B

Reply:
In which case, I don't  this sentence means `B`: 

> in a `facetStats` object that contains, per facet, the numeric `min` and `max` values of the hits returned by the search query.

Reply:
In that case, suggesting:

```suggestion
When using the `facets` parameter, the distributed facets that contain some numeric values are displayed in a `facetStats` object that contains, per facet, the numeric `min` and `max` values for that facet of all candidate documents matching the search query.
```

I'm not sure about the "remaining candidates" phrasing, as we return the min-max of *all* candidates (and same for facet distribution?)

Reply:
I like your suggestion!